dfa/src/main/scala/dfa/Program.scala

```diff
@@ -12,8 +12,8 @@ val testDFA = {
   DFA(
     states = Set(s0, s1),
     transitions = Set(t11, t12, t01, t02),
-    start = s0,
-    accept = Set(s0)
+    startState = s0,
```

Comment:
it's somewhat strange tthat the assignment assumed the implementation was using variable names that were different from the ones in the requirements, and so you have to change the program file to use your own specific variables.

It feels like you could generalize this to APIs in general, and why many of them require redundant-seeming public functions (like GetThing() ) and keeping the internal variables private.